### PR TITLE
Change training hours to military specialist units

### LINKED ISSUE #146: Change specialist unit training times from 12hr to 9hr
### Expected Behavior
Military specialist units should take only 9 hours to train, instead of 12 hours.

All other units (elite, spies and mages) should still take 12 hours to train.

### Current Behavior
All units take 12 hours to train currently.

### Context
Specialist units are the fields 'military_unit1' and 'military_unit2' on the dominions table. This is the same for every race. (fyi units 3 and 4 are elite units, but that's out of the scope for this issue).

Code change should probs go in the TrainActionService class.

### PR DESCRIPTION
**Pull request by danielsouzaaf**
Closes #146.

### CODE REVIEW

File: src/Services/Dominion/Actions/Military/TrainActionService.php [hunk 1]
```diff
@@ -119,10 +118,12 @@ public function train(Dominion $dominion, array $data): array
                     continue;
                 }
 
+                dd($unitType);
```
Comment (Hodgy) [inline src/Services/Dominion/Actions/Military/TrainActionService.php]: Probably want to remove this.

Comment (danielsouzaaf) [reply src/Services/Dominion/Actions/Military/TrainActionService.php]: Ooops! Removed.

File: src/Services/Dominion/Actions/Military/TrainActionService.php [hunk 2]
```diff
@@ -107,7 +107,6 @@ public function train(Dominion $dominion, array $data): array
             $existingQueueRows = DB::table('queue_training')
                 ->where([
                     'dominion_id' => $dominion->id,
-                    'hours' => 12,
```
[the remark this answers is not in the document]
Comment (WaveHack) [reply src/Services/Dominion/Actions/Military/TrainActionService.php]: Yeah this one is needed so it doesn't insert units multiple times at different hours.

[the remark this answers is not in the document]
Comment (danielsouzaaf) [reply src/Services/Dominion/Actions/Military/TrainActionService.php]: I'll make the changes tomorrow! Thanks for the patience

**Comment by WaveHack** [comment]
Still the same issue. Just verified my findings.

Steps to reproduce:

1. Start a fresh local setup and register a dominion (in my case human race).
2. Give yourself some ore manually
3. Train 2 spearman, observe in db they get inserted at hour 9. OK
4. Train 1 knight, observe in db they get inserted at hour 12, OK
5. Either subtract 3 hours from both records in queue_training or tick the game 3 times using `bin/artisan game:tick`
6. Train 1 spearman, observe in db it got inserted at hour 9, OK, but a wild knight also appears out of nowhere at hour 12 while not having trained this, not OK

**Comment by WaveHack** [comment]
I just fixed it. Thanks for your effort.

**Pull request closed without merging**